go/constatnts.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+const DB_FILE_PATH = "../scheduler.db"
```

Comment:
Обычно проекты запускают из корня проектов, так что и пути должны идти относительно корня
```suggestion
const DB_FILE_PATH = "./scheduler.db"
```

Reply:
теперь это не зависит от папки запуска